Gruntfile.js

```diff
@@ -72,6 +74,13 @@ module.exports = function (grunt) {
     });
   });
 
+  // load local Grunt tasks
+  grunt.loadTasks('tasks');
+
+  grunt.registerTask('lint',
+  'Alias for eslint tasks',
+  ['eslint']);
```

Comment:
I thought maybe if we needed to add say, jscs or some other thing to the lint, this would be more readable. Should I change it? 
@vladikoff